src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSelector.js

```diff
@@ -140,7 +140,7 @@ class MoneyRequestParticipantsSelector extends Component {
      * @param {Object} option
      */
     addSingleParticipant(option) {
-        this.props.onAddParticipants([option]);
+        this.props.onAddParticipants([{login: option.login, isPolicyExpenseChat: option.isPolicyExpenseChat, reportID: option.reportID, selected: true}]);
```

Comment:
Related to above, options in MoneyRequestParticipantsSelector is constructed from `OptionsListUtils.createOption` which doesn't provide `isOwnPolicyExpenseChat`, but only `isPolicyExpenseChat`

Reply:
Also, that is why in IOU.requestMoney, we check for both `isPolicyExpenseChat` and `isOwnPolicyExpenseChat`
https://github.com/Expensify/App/blob/0813e2ca1476635a8324cbd9f998424663a33128/src/libs/actions/IOU.js#L266

Reply:
Can we unify this? Drop the `isOwnPolicyExpenseChat` it's useless, we can just relay on `isPolicyExpenseChat` (other chats than isOwnPolicyExpenseChat does not have an option to be even selected).

Reply:
But `isPolicyExpenseChat` is only available if we select the participant from participant selector as explained [here](https://github.com/Expensify/App/pull/17964#discussion_r1216412187).

I was thinking to drop both `isOwnPolicyExpenseChat` and `isPolicyExpenseChat` and just rely on `reportID`. If `reportID` exists, we know that it is a policy expense chat. So, the above logic will be:

```
const isPolicyExpenseChat = Boolean(participant.reportID)
```

If in the future, other chat type able to request money too, we can just get the report from `reportID` and get the `isOwnPolicyExpenseChat` attribute (`reports[participant.reportID].isOwnPolicyExpenseChat`). Or we can actually do that now, we already have the `chatReports` onyx in `IOU.js`

```
const isPolicyExpenseChat = chatReports[participant.reportID].isOwnPolicyExpenseChat
```

Reply:
In the cases where we don't have isPolicyExpenseChat as a prop (not an option item). What do we have, is it a report? If so can we just call  `ReportUtils.isPolicyExpenseChat`?

Let's not relay on reportID because it's unclear. isPolicyExpenseChat seems the best fit here.

Reply:
As we are talking about `isOwnPolicyExpenseChat`, let's move the discussing above.

Reply:
Resolving here as `isOwnPolicyExpenseChat` is removed.